src/ci_cd.tex

```diff
@@ -91,16 +91,16 @@ \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Tes
 %%%%%%%% RIGHT COLUMN
 
 \section{\includegraphics[height=\fontcharht\font`\S]{img/general/slash.png} Build library}
-Leveraging pre-built components and artifacts to expedite software development,
+Leverages pre-built components and artifacts to expedite software development,
```

Comment:
This and the next section having the same content inside @jorgepiloto , is this expected?

Reply:
What do you mean @Revathyvenugopal162?

Reply:
Oh Now I understand... Yeah something went wrong here @jorgepiloto - please have a look at it

Reply:
Looks like it should have this text--which was on the poster here: https://scipy2023.pyansys.com/ci_cd.pdf
Guarantees the creation of components and artifacts for efficient software distribution. Verifies that the artifacts include the proper metadata. 
(Hope I didn't cause the problem!)

Reply:
No worries @PipKat! :) we had @jorgepiloto re-write it. It was a mistake on our side in fact. He just opened a PR and we merged it. Thanks anyway!